x-pack/legacy/plugins/apm/public/services/__test__/callApi.test.ts

```diff
@@ -158,5 +161,34 @@ describe('callApi', () => {
         expect(http.get).toHaveBeenCalledTimes(1);
       });
     });
+
+    describe('when posting', () => {
```

Comment:
Perhaps also a test for GET request since they are handled differently?

Reply:
Done!